java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -4170,6 +4203,14 @@ private static native long substringColumn(long columnView, long startColumn, lo
    */
   private static native long stringReplace(long columnView, long target, long repl) throws CudfException;
 
+  /**
+   * Native method to replace target strings by corresponding repl strings.
+   * @param columnView native handle of the cudf::column_view being operated on.
+   * @param targets handle of column containing the strings being searched.
+   * @param repls handle of column containing the strings to replace (can optionally contain a single string)
+   */
+  private static native long stringReplaceMulti(long columnView, long targets, long repls) throws CudfException;
```

Comment:
All 3 parameters are column view.
```suggestion
  private static native long stringReplaceMulti(long inputCV, long targetsCV, long replsCV) throws CudfException;
```